react/basics.md

```diff
@@ -64,16 +64,14 @@ At the top of your component file, you'll need to import the following packages:
 
 import React from 'react'
 import PropTypes from 'prop-types'
-import { connect } from 'react-redux'
-import { compost } from 'redux'
 ```
 
-Depending on what type of component you're building, you many need to import additional packages. For now `react`, `prop-types`, `react-redux`, and `redux` represent the bare minimum you'll need to get started.
+Depending on what type of component you're building, you many need to import additional packages. For now `react` and `prop-types` represent the bare minimum you'll need to get started with your first React component. We'll be adding more packages as we work through the component lifecycle.
```

Comment:
The last sentence is a bit confusing- is it referring to the React [component lifecycle](https://reactjs.org/docs/react-component.html#the-component-lifecycle) or the process of creating a component?